### PR TITLE
BaSP-MR-28: Test endpoints from member

### PR DESCRIPTION
**Pull request by moradpablo**
Fixes made since last week and test 3 member endpoints (get, get by id and post) changed some member files to be easier to read. Seed and test from member created  👍 
https://trello.com/b/nE1jqSay/basp-m2023-baru

### CODE REVIEW

**Review by CristianLotorto — APPROVED** [review]
Everything works fine. Nice work with tests!

File: src/tests/members.test.js [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+    expect(response.body.name).toBeDefined();
+    expect(response.body.dni).toBeDefined();
+    expect(response.body.city).toBeDefined();
+    expect(response.body.dob).toBeDefined();
+    expect(response.body.zip).toBeDefined();
+    expect(response.body.isActive).toBeDefined();
+    expect(response.body.membership).toBeDefined();
+    expect(response.body.email).toBeDefined();
+    expect(response.body.password).toBeDefined();
+  });
```
Comment (maurojjzz) [inline src/tests/members.test.js:81]: as i recommendation i would say to separate the lines with the method toBeDefined() onto another test and leave the 'should create a member with status 201' until line 70

Comment (maurojjzz) [reply src/tests/members.test.js:81]: and the test with the lines with .toBeDefined() could be named as 'should have all the required fields' or something like that

Comment (maurojjzz) [reply src/tests/members.test.js:81]: that will improve organization and readability.

**Review by maurojjzz — APPROVED** [review]
i made a suggestion but other than that everything seems fine

**Review by BareaMartin — APPROVED** [review]

**Pull request merged**